homeassistant/components/openrgb/config_flow.py

```diff
@@ -0,0 +1,81 @@
+"""Config flow for the OpenRGB integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from openrgb import OpenRGBClient
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PORT
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv
+
+from .const import CONNECTION_ERRORS, DEFAULT_CLIENT_NAME, DEFAULT_PORT, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_NAME): str,
```

Comment:
We only allow user settable names for helpers

Reply:
I understand. Can you please read [this comment](https://github.com/home-assistant/core/pull/153373#issuecomment-3365487248), and share on whether you have any suggestions on how to handle it?